src/app/modules/user/__tests__/user.controller.spec.ts

```diff
@@ -436,27 +432,36 @@ describe('user.controller', () => {
   })
 
   describe('handleUpdateUserLastSeenFeatureUpdateDate', () => {
-    const MOCK_DATE = new Date()
     const MOCK_REQ = expressHandler.mockRequest({
-      body: { latestLastSeenFeatureUpdateDate: MOCK_DATE },
       session: {
         user: {
           _id: VALID_SESSION_USER_ID,
         },
       },
     })
+    // Mock implementaion of Date.now() due to slight difference in Date.now() execution in test and when
+    // Date.now() in updateUserLastSeenFeatureUpdateDate function is executed
+    jest
+      .spyOn(global.Date, 'now')
+      .mockReturnValue(new Date('2022-07-14T11:01:58.135Z').valueOf())
+    const MOCK_DATE = new Date('2022-07-14T11:01:58.135Z')
```

Comment:
can probably create `MOCK_DATE` first then use that in the spy haha

Reply:
we also have a package specifically for mocking all `Date` constructor calls in tests called `MockDate`, might want to use it over this implementation

Reply:
I will check it and update accordingly

Reply:
When I checked through the codebase, I was not able to find any util file that mocks the Date constructor. All the spec files that mock the Date constructor calls did it by calling `jest.spyOn(Date, now).mockReturnValue()`. Could I ask where I can find the package which mocks the Date constructor calls?

Reply:
https://github.com/opengovsg/FormSG/blob/develop/src/app/models/__tests__/user.server.model.spec.ts#L1